### ISSUE TITLE
Drag and Drop From Filesystem

### ISSUE THREAD

**Issue by zolrath** [Bumblebee/Bumblebee#89]
Selenium itself does not have support for Drag and Drop operations from the file system onto the page.
This would be useful for things such as testing dragging an image onto a drag and drop image uploader.

There are various workarounds which involve injecting an html tag into the page pointing to the local file and then dragging that element onto the dropzone.

http://stackoverflow.com/a/23469605/949692
https://github.com/Kuniwak/html-dnd

It would be nice if there was a convenient API for dragging and dropping from the filesystem.

On a more base level -- I'm not seeing anything around uploading files to <input type="file"> elements in the wiki?

**Comment by toddmeinershagen** [comment]
@zolrath Not sure if this feature is still needed, but I am reaching out to see.  Right now I am marking it as a question.  If I hear back we can add it to the backlog for v2.2.  Hope it helps.

**Comment by zolrath** [comment]
Yes, having robust drag and drop support would be great for a number of interactions on most of the sites I test. Drag and drop between elements on the screen/drag and drop files from the file system would be great!
